pkg/token/token_test.go

```diff
@@ -48,8 +48,8 @@ func assertSTSError(t *testing.T, err error) {
 var (
 	now        = time.Now()
 	timeStr    = now.UTC().Format("20060102T150405Z")
+	validURL   = fmt.Sprintf("https://sts.amazonaws.com/?action=GetCallerIdentity&X-Amz-Credential=ASIABCDEFGHIJKLMNOPQ%%2F20191216%%2Fus-west-2%%2Fs3%%2Faws4_request&x-amz-signedheaders=x-k8s-aws-id&x-amz-expires=60&x-amz-date=%s", timeStr)
```

Comment:
Can you also add a test without the query param `X-Amz-Credential` so we ensure older clients are still valid?

Reply:
Not sure what you mean.  The presigned URL issued by the client has always contained that query parameter; we were just never making use of it in the past.  There's no older behavior to test AFAIK.

Reply:
ah my bad, I thought it wasn't being included by the client. Thanks for the fix in the test